src/main/java/io/adobe/cloudmanager/Metric.java

```diff
@@ -67,35 +69,35 @@ public interface Metric {
   /**
    * Represents the severity of the metric.
    */
+  @Getter
   enum Severity {
     CRITICAL("critical"),
```

Comment:
Since the string is the lowercased version of the constant name, couldn't you just use case-insensitive matching and otherwise use a lowercased call to name()?